web/src/org/purefn/kurosawa/web/server.clj

```diff
@@ -57,6 +57,16 @@
                                  (str dir "/immutant.log"))))
 
 
+(def default-worker-threads
+  "Default worker thread-count for the web server, which is the same as in Immutant,
+   that is (* 8 IO-threads) where IO-threads is available processor count."
+  (* 8 (.availableProcessors (Runtime/getRuntime))))
```

Comment:
I'm not sure what `.availableProcessors` will return in our current k8s runtime (Docker) but it may not be quite what we want.  Still, I think it's a sensible default, we may just want to remember to pass an explicit override in application configs.

Reply:
I agree the app using Kurosawa should explicitly specify the params. This default only mimics the Immutant default behaviour.